secondary-server-1/src/main/java/com/replicated_log/secondary_server_1/service/ItemService.java

```diff
@@ -0,0 +1,16 @@
+package com.replicated_log.secondary_server_1.service;
+
+import com.replicated_log.secondary_server_1.model.Item;
+
+import java.util.Set;
+
+public interface ItemService {
+
+    boolean addItem(Item item);
+
+    Set<Item> getItems();
+
+    Set<Item> getItemsForDemo();
+
+    void simulateProcessing(int seconds);
```

Comment:
I can see , according to the signature of the method simulateProcessing , number 5 is seconds
what is the method simulateProcessing ?

Reply:
simulateProcessing is the method which was used to test that the replication is blocking, (the delay/sleep on the Secondary was introduced in this method). It was one of the Project Requirements.